media_driver/linux/common/os/i915/include/mos_bufmgr.h

```diff
@@ -308,8 +308,6 @@ int mos_get_reset_stats(struct mos_linux_context *ctx,
                   uint32_t *active,
                   uint32_t *pending);
 
-int mos_get_slice_mask(uint32_t slice_count);
```

Comment:
Looks to be a cleanup which we should merge right now. Can you provide a separate PR for this?

Reply:
yes.

Reply:
Done with PR https://github.com/intel/media-driver/pull/480